Vic2ToHoI4/Source/OutHoi4/IntelligenceAgencies/OutIntelligenceAgencies.cpp

```diff
@@ -0,0 +1,17 @@
+#include "OutIntelligenceAgencies.h"
+#include "OSCompatibilityLayer.h"
+#include "OutIntelligenceAgency.h"
+#include <fstream>
+
+
+
+void HoI4::outputIntelligenceAgencies(const IntelligenceAgencies& intelligenceAgencies, const std::string& outputName)
+{
+	Utils::TryCreateFolder("output/" + outputName + "/common/intelligence_agencies/");
```

Comment:
reworked Try functions still return bool. Not sure if you updated commons yet, though.